test/config/postgres.config

```diff
@@ -0,0 +1,6 @@
+driver="PostgreSQL Unicode"
+host="localhost"
+port="5432"
+dbname="fdw_tests"
```

Comment:
Just two random thoughts, not to answer here:
- hive will be lots of fun
- some other config files might require encryption.